sdk/documenttranslation/tests.yml

```diff
@@ -0,0 +1,13 @@
+trigger: none
+
+jobs:
+  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
```

Comment:
will this add a pipeline for nightly test? will the samples start running?
I just wonder how the cleanup of the target container will work

Reply:
No, only the ci pipeline will be created for recorded tests. We still need to figure out how we'll do creation/tear down of live resources. I'm going to look into that today a bit.